Sources/SmithyIdentity/AWSCredentialIdentityResolvers/StaticAWSCredentialIdentityResolver.swift

```diff
@@ -24,3 +30,11 @@ public struct StaticAWSCredentialIdentityResolver: AWSCredentialIdentityResolver
         return credentials
     }
 }
+
```

Comment:
The `ClientConfigDefaultIdentityResolver` extension below marks this resolver as a client config default if the credentials are empty, something that the new initializer above does, but a real customer should never do.